Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -543,6 +567,7 @@ final class LayerTests: XCTestCase {
         ("testRNN", testRNN),
         ("testFunction", testFunction),
         ("testBatchNorm", testBatchNorm),
-        ("testLayerNorm", testLayerNorm)
+        ("testLayerNorm", testLayerNorm),
+        ("testLSTM", testLSTM)
```

Comment:
Place this between "testRNN" and "testFunction" like how the methods are ordered.